### PR TITLE
[feat] uistate 연결

### PR DESCRIPTION
**Pull request by ruh0n**
## :man_shrugging: Description

- [x] uistate를 composable 에서 사용하도록 연결

### CODE REVIEW

**Comment by ruh0n** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/boostcampwm-2024/and01-dreamDiary/101?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#101** <a href="https://app.graphite.dev/github/pr/boostcampwm-2024/and01-dreamDiary/101?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#100** <a href="https://app.graphite.dev/github/pr/boostcampwm-2024/and01-dreamDiary/100?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ruh0n and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by flashdragon — APPROVED** [review]
굳

**Comment by Cheonilyeong** [comment]
LGTM 😎🔥🔥

File: feature/diary/src/main/java/com/boostcamp/dreamteam/dreamdiary/feature/diary/home/tabcalendar/DiaryCalendarTab.kt [hunk 1]
```diff
@@ -8,11 +8,21 @@ import androidx.compose.ui.tooling.preview.Preview
...
+import com.boostcamp.dreamteam.dreamdiary.feature.diary.models.diariesPreview
+import java.time.YearMonth
 
 @Composable
-internal fun DiaryCalendarTab(modifier: Modifier = Modifier) {
+internal fun DiaryCalendarTab(
+    modifier: Modifier = Modifier,
+    state: DiaryHomeTabCalendarUIState = DiaryHomeTabCalendarUIState(),
+) {
+    val (yearMonth) = state
```
Comment (easyhak) [inline feature/diary/src/main/java/com/boostcamp/dreamteam/dreamdiary/feature/diary/home/tabcalendar/DiaryCalendarTab.kt:19]: 이렇게 하기로 한 것인가요?

**Pull request merged**